configure

```diff
@@ -17888,6 +17889,11 @@ if test "`uname`" = "NetBSD"; then
 	NETBSD_LINTFLAGS='"-D__RENAME(x)=" -D_NETINET_IN_H_'
 
 fi
+
+if test "`uname -o`" = "GNU/Linux"; then
+	# splint cannot parse modern c99 header files
+	GCC_DOCKER_LINTFLAGS='-syntax'
+fi
```

Comment:
These changes need to go to `configure.ac` instead. `configure` is auto-generated (also) by the maintainers based on that file.

Reply:
done